hangman/README.md

```diff
@@ -1,3 +1,12 @@
 8k students only:
 
 Describe how to run the multinode app.
+
+I have modified the hangman module to spawn a process registered as :foo. This process waits for :new_game requests and responds by creating a new game server and passing its pid. This pid is then used by the human_player module to interact with the game server.
```

Comment:
An alternative to this is to use a remote procedure call from the client to the server, calling new_game. This would then return the pid (on the server) to the client.

Just an observation